### PR TITLE
feat: CQDG-135 Add dictionnary in HpoConditionCell and getColumnTitleOrTooltip in ExpandableTable

### PR DESCRIPTION
**Pull request by atoulous**
# feat: CQDG-135 Add dictionnary in HpoConditionCell and getColumnTitleOrTooltip in ExpandableTable

- closes #TICKET_NUMBER

## Description

https://ferlab-crsj.atlassian.net/browse/CQDG-135

Acceptance Criterias

## Validation

- [ ] Storybook add or modified
- [ ] version Update in package.json and Release.md
- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

## QA

Steps to validate
Url (storybook, ...)
...

### CODE REVIEW

File: packages/ui/src/components/tables/ExpandableTable/index.tsx [hunk 1]
```diff
@@ -23,9 +27,28 @@ const ExpandableTable = ({
...
+            ) : (
+                title
+            );
+        title = column.tooltip ? <Tooltip title={column.tooltip}>{title}</Tooltip> : title;
+        return { ...column, title } as ProColumnType;
+    };
+
     return (
         <>
-            <Table dataSource={(dataSource || []).slice(0, sliceNum)} {...tableProps} />
```
Comment (atoulous) [inline packages/ui/src/components/tables/ExpandableTable/index.tsx:28]: in this order it looks like dataSource was override by tableProps.dataSource

File: packages/ui/src/components/tables/ExpandableTable/index.tsx [hunk 2]
```diff
@@ -23,9 +27,28 @@ const ExpandableTable = ({
     const showButton = dataTotalLength > nOfElementsWhenCollapsed;
     const hiddenNum = dataTotalLength - sliceNum;
 
+    const getColumnTitleOrTooltip = (column: ProColumnTypes) => {
```
Comment (atoulous) [inline packages/ui/src/components/tables/ExpandableTable/index.tsx:30]: same algo as getTitleByType in ProTable

File: packages/ui/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ferlab/ui",
-    "version": "7.13.7",
+    "version": "7.13.8",
```
Comment (celinepelletier) [inline packages/ui/package.json:3]: devrait pas y avoir le bump de version dans le package-lock aussi ?

**Review by celinepelletier — APPROVED** [review]

File: packages/ui/package.json (hunk 3 above)
Comment (atoulous) [reply packages/ui/package.json:3]: non c'est quand tu installes une nouvelle version d'un lib-dépendance ca. Donc coté portail oui le package-lock va évoluer

**Pull request merged**